utils/sub8_ros_tools/sub8_ros_tools/msg_helpers.py

```diff
@@ -113,14 +113,14 @@ def numpy_matrix_to_quaternion(np_matrix):
     return geometry_msgs.Quaternion(*np_quaternion)
 
 
-def numpy_pair_to_pose(np_translation, np_rotation_matrix):
```

Comment:
Oh no! Just ran into an issue here. We should probably switch these back or switch the other code that was dependent on the ordering here.